suites/reef/rados/tier-2_rados_test-pg-split-merge.yaml

```diff
@@ -138,6 +139,51 @@ tests:
           - rpool_3
           - ecpool_2
 
+  - test:
+      name: Test Reads Balancer
+      module: test_reads_balancer.py
+      desc: Testing reads balancer offline tool functionality in RHCS
+      polarion-id: CEPH-83576078
```

Comment:
Do we have a separate polarion ID for this? or the same ID covers both the scenarios?

Reply:
My bad, there is a separate test case, will update.

Reply:
Updated!